accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -6,6 +6,23 @@ Status: Draft
 
 ## CHANGELOG
 
+2020.01.29
+  - **CHANGE** Relax the exhaustiveness check on switches.
+  - Specify the type of throw expressions.
+  - Specify the override inference exception for operator==.
+  - **CHANGE** Specify that instantiate to bounds uses `Never` instead of `Null.
```

Comment:
missed trailing "tick" after `Null` on this line!

Reply:
Fixed, thanks!